doc/releases/migration-guide-4.0.rst

```diff
@@ -538,10 +552,17 @@ JWT (JSON Web Token)
 ====================
 
 * By default, the signature is now computed through PSA Crypto API for both RSA and ECDSA.
-  The newly-added :kconfig:option:`CONFIG_JWT_USE_LEGACY` can be used to switch
-  back to previous libraries (TinyCrypt for ECDSA and Mbed TLS for RSA).
-  The conversion to the PSA Crypto API is being done in preparation for the
-  deprecation of TinyCrypt. (:github:`78243` and :github:`43712`)
+  (:github:`78243`). The conversion to the PSA Crypto API is part of the adoption
```

Comment:
Mmmm albeit what you suggest is the "standard" way of adding github references, that works fine if the reference is at the end of the sentence, which is not the case here. I prefer to keep that full stop mark and remove the previous one because that reference is related to the previous sentence, not the one that is starting.

Reply:
My point here is that you have two dots, without sentence between them.